base/cpuid.jl

```diff
@@ -64,7 +64,11 @@ const ISAs_by_family = Dict(
     "powerpc64le" => [
         # We have no way to test powerpc64le features yet, so we're only going to declare the lowest ISA:
         "power8" => ISA(Set{UInt32}()),
-    ]
+    ],
+    "riscv64" => [
+        # We have no way to test riscv64 features yet, so we're only going to declare the lowest ISA:
+        "riscv64" => ISA(Set{UInt32}()),
```

Comment:
This seems not great to me. Anything with enough memory to run julia almost certainly is RVA20 at least. Integer multiplication, atomics, and floating point are all going to be in pretty much everything other than the simplest micro-controllers. As such, I think it's reasonable to start by requiring a decent number of extensions and adding support for lower end chips once we support testing for features.

Reply:
For anyone reading this, this module is mainly needed for BinaryBuilder stuff, it has nothing to do with Julia's own compilation and optimisations.